### PR TITLE
Allow parameters to be used in property configuration

### PR DESCRIPTION
**Pull request by mcgilman**
This PR should be used on top of #3606. It supports retrieving parameters and allowing them to be selected via Ctrl-Space and offers syntax highlighting. This single PR addresses a number of JIRAs. Please see the commit log.

Update:
#3606 has been merged and this PR has been rebased. It is ready for review

### CODE REVIEW

**Comment by scottyaslan** [comment]
Reviewing...

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/nfeditor/jquery.nfeditor.js [hunk 1]
```diff
@@ -236,6 +234,28 @@
             });
         },
 
+        /**
+         * Sets the size of the editor.
```
Comment (rfellows) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/nfeditor/jquery.nfeditor.js]: Looks like a `CTRL-C`, `CTRL-V` here. Should update this to reflect what the function actually does 😄

**Comment by scottyaslan** [comment]
@mcgilman I noticed that when a user is editing a property table cell and resizes the editor that the tooltip for the EL/Param is off the screen:

Screen Shot 2019-08-30 at 11 48 45 AM

I think we can address this in a follow on PR. I created https://issues.apache.org/jira/browse/NIFI-6605 to track this issue.

**Comment by scottyaslan** [comment]
@mcgilman Thanks! This has been merged to master.

**Pull request closed without merging**